ci/scripts/image_scripts/build_ipa_on_centos.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Set execution parameters to:
+# Fail whenever any command fails
+set -eu
+
+# Configurable options
+DISABLE_UPLOAD="${DISABLE_UPLOAD:-false}"
+IPA_BUILD_WORKSPACE="${IPA_BUILD_WORKSPACE:-/tmp/dib}"
+RT_UTILS="${RT_UTILS:-/tmp/utils.sh}"
+RT_URL="${RT_URL:-https://artifactory.nordix.org/artifactory}"
+IPA_BUILDER_PATH="${IPA_BUILDER_PATH:-ironic-python-agent-builder}"
+IPA_IMAGE_NAME="${IPA_IMAGE_NAME:-ironic-python-agent}"
+IPA_IMAGE_TAR="${IPA_IMAGE_NAME}.tar"
+IPA_BUILDER_REPO="${IPA_BUILDER_REPO:-https://github.com/Nordix/ironic-python-agent-builder.git}"
+IPA_BUILDER_BRANCH="${IPA_BUILDER_BRANCH:-master}"
+IPA_REPO="${IPA_REPO:-https://github.com/Nordix/ironic-python-agent}"
+IPA_REPO_CONF="${IPA_IMAGE_NAME} git /tmp/ironic-python-agent ${IPA_REPO}"
+IPA_REPO_CONF_PATH="${IPA_BUILDER_PATH}/dib/ironic-python-agent-ramdisk/source-repository-ironic-python-agent"
+IPA_BASE_OS="${IPA_BASE_OS:-centos}"
+IPA_BASE_OS_RELEASE="${IPA_BASE_OS_RELEASE:-8}"
+# Install required packages
+sudo apt install --yes python3-pip python3-virtualenv qemu-utils
+
+# Create the work directory
+mkdir --parents "${IPA_BUILD_WORKSPACE}"
+cd "${IPA_BUILD_WORKSPACE}"
+
+# Pull IPA builder repository
+git clone --single-branch --branch "${IPA_BUILDER_BRANCH}" "${IPA_BUILDER_REPO}"
+
+# Change the IPA repository address that is used during the build process
+echo -n "${IPA_REPO_CONF}" > "${IPA_REPO_CONF_PATH}" 
+
+# Generate the IPA image identifier string
+cd ironic-python-agent-builder
+# IDENTIFIER is the git commit of the HEAD and the ISO 8061 UTC timestamp
+IPA_IDENTIFIER="$(date --utc +"%Y-%m-%dT%H:%MZ")-$(git rev-parse HEAD)"
+cd ..
+
+# Install the cloned IPA builder tool
+virtualenv venv
+# shellcheck source=/dev/null
+source "./venv/bin/activate"
+python3 -m pip install --upgrade pip
+python3 -m pip install "${IPA_BUILDER_PATH}"
+
+# Build the IPA initramfs and kernel images
+ironic-python-agent-builder --output "${IPA_IMAGE_NAME}" \
+    --release "${IPA_BASE_OS_RELEASE}" --branch "master" "${IPA_BASE_OS}" --verbose
```

Comment:
```suggestion
ironic-python-agent-builder --output "${IPA_IMAGE_NAME}" \
    --release "${IPA_BASE_OS_RELEASE}" -e dynamic-login -e journal-to-console -e devuser -e openssh-server --branch "master" "${IPA_BASE_OS}" --verbose
```
Can you please add these elements as per the request of @maelk ? If I remember correct,  we can't use -e flag once and give a list of elements with comma in between. Maybe we can try that by surrounding the list with double quotes. If not the last option, as an example I give above.

Reply:
I have added these to the script, I have tested the -e option and it looks like we have to use multiple
 -e options to pass multiple additional element names.

Reply:
ok, that should be fine.